### PR TITLE
fix avdl names for chat since it's not released yet

### PR DESCRIPTION
**Pull request by songgao**
r? @mmaxim @maxtaco Thanks!

### CODE REVIEW

File: go/chat/sender_test.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func TestNonblockChannel(t *testing.T) {
 			TopicType: 0,
 			TopicID:   []byte{0},
 		},
-		TLFMessage: chat1.MessageBoxed{
+		TlfMessage: chat1.MessageBoxed{
```
Comment (patrickxb) [inline go/chat/sender_test.go]: I prefer TLFMessage on the go side here.

Comment (songgao) [inline go/chat/sender_test.go]: For `TLFMessage`: keybase/node-avdl-compiler#10 PTAL Thanks!

**Comment by keybase-ci-visdiff** [comment]
The commits abed0dfbd0c1...9723ad17b51b introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Chat_Input-Emoji_Open** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480956744206-abed0dfbd0c1-9723ad17b51b-linux/Chat_Input-Emoji_Open-abed0dfbd0c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480956744206-abed0dfbd0c1-9723ad17b51b-linux/Chat_Input-Emoji_Open-9723ad17b51b.png)  
   Chat_Input-Emoji_Open comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits abed0dfbd0c1...9723ad17b51b introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Chat_Input-Emoji_Open** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480961515441-abed0dfbd0c1-9723ad17b51b-linux/Chat_Input-Emoji_Open-abed0dfbd0c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480961515441-abed0dfbd0c1-9723ad17b51b-linux/Chat_Input-Emoji_Open-9723ad17b51b.png)  
   Chat_Input-Emoji_Open comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits abed0dfbd0c1...f6595a324231 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Chat_Input-Emoji_Open** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480963112667-abed0dfbd0c1-f6595a324231-linux/Chat_Input-Emoji_Open-abed0dfbd0c1.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480963112667-abed0dfbd0c1-f6595a324231-linux/Chat_Input-Emoji_Open-f6595a324231.png)  
   Chat_Input-Emoji_Open comparison
</summary>

**Pull request merged**